stm/stm.ml

```diff
@@ -65,7 +65,7 @@ end
 
 (* During interactive use we cache more states so that Undoing is fast *)
 let interactive () =
-  if !Flags.ide_slave || !Flags.print_emacs || not !Flags.batch_mode then `Yes
+  if !Flags.ide_slave || not !Flags.batch_mode then `Yes
```

Comment:
In fact this test should be reduced to just `not !Flags.batch_mode`, `ide_slave` is another flag that should be removed once we tweak Stm.add.